Apps/Sandcastle/CesiumSandcastle.js

```diff
@@ -810,6 +811,14 @@ require({
         }
     }
 
+    registry.byId('buttonNew').on('click', function() {
+        loadFromGallery(newDemo);
+        var demoSrc = newDemo.name + '.html';
+        if (demoSrc !== window.location.search.substring(1)) {
```

Comment:
To avoid pushing the demo to the history if the Hello World example was already open. Otherwise, if you're already on hello world and you click on new again, your history would now have two hello world pages in its stack.